src/components/metadata/AuthorsPage.tsx

```diff
@@ -13,46 +13,19 @@
 import AddedIcon from '@manuscripts/assets/react/AddedIcon'
 import AuthorPlaceholder from '@manuscripts/assets/react/AuthorPlaceholder'
 import ContributorDetails from '@manuscripts/assets/react/ContributorDetailsPlaceholder'
-import InvitationPlaceholder from '@manuscripts/assets/react/InvitationPlaceholder'
 import React from 'react'
-import { FormattedMessage } from 'react-intl'
 import styled from 'styled-components'
 
 const AddAuthorsMessage = () => (
```

Comment:
I believe all these messages are not in use, and we can get rid of them.

We are not using the `AddAuthorsModalContainer` and all of its related components, including `AddAuthorsModal`, `AddAuthorsSidebar`, `AddAuthorsPage`, and other related components.

Is there a reason to keep them?

Reply:
There will be a separate ticket to move the author modals to style-guide, and delete unused items.